airflow/www_rbac/static/webpack.config.js

```diff
@@ -0,0 +1,128 @@
+/**
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements. See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership. The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License. You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+
+const webpack = require('webpack');
+const path = require('path');
+const ManifestPlugin = require('webpack-manifest-plugin');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+// Input Directory (airflow/www_rbac)
+// noinspection JSUnresolvedVariable
+const STATIC_DIR = path.resolve(__dirname, './');
+
+// Output Directory (airflow/www_rbac/static/dist)
+// noinspection JSUnresolvedVariable
+const BUILD_DIR = path.resolve(__dirname, './dist');
+
+const config = {
```

Comment:
Does this generate source maps? If not please can you enable that - even for "prod" builds please.

Reply:
will enable source maps. good point!!